buildlib/pr/main.yml

```diff
@@ -432,6 +432,21 @@ stages:
         test_perf: 0
         container: ubuntu2204_ib
         asan_check: yes
+    - template: tests.yml
+      parameters:
+        name: roce
+        demands: ucx_roce -equals yes
+        test_perf: 0
+        container: ubuntu2204_ib
+        asan_check: yes
+    - template: tests.yml
+      parameters:
+        name: roce_proto_disable
+        demands: ucx_roce -equals yes
+        test_perf: 0
+        proto_enable: no
+        container: ubuntu2204_ib
+        asan_check: yes
```

Comment:
IMO this is not needed - we can remove to reduce testing time
@brminich WDYT?

Reply:
maybe we can leave ASAN but remove valgrind for proto disable instead?

Reply:
Done.